modules/module-extras.php

```diff
@@ -83,3 +83,28 @@ function jetpack_widgets_add_suffix( $widget_name ) {
 	);
 }
 add_filter( 'jetpack_widget_name', 'jetpack_widgets_add_suffix' );
+
+add_action( 'blog_privacy_selector', 'jetpack_priv_notice_privacy_selector' );
+
+/**
+ * Echos notice directing site owners to Jetpack's Private Site feature.
+ */
+function jetpack_priv_notice_privacy_selector() {
+	?>
+	<p>
+	<?php
+
+		wp_kses(
+			printf(
+				/* translators: URL to the Jetpack dashboard. */
+				__( 'You can also make your site completely private by allowing only registered users to see it. <a href="%s">Go to Private Site settings</a>.', 'jetpack' ),
```

Comment:
This part is annoying. I know we've come back and forth on it:
https://github.com/Automattic/jetpack/pull/10333/files#r270569491

It's now easier to translate, but it generates a phpcs error that should actually have blocked the commit:

```
----------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------
 100 | ERROR | All output should be run through an escaping function (see the Security
     |       | sections in the WordPress Developer Handbooks), found '__'.
     |       | (WordPress.Security.EscapeOutput.OutputNotEscaped)
----------------------------------------------------------------------------------------------
```

That error does not make much sense here though imo, since we run the whole thing through `wp_kses`. I'd suggest adding `// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped` so it does not error out.

Reply:
Fixed in [0f9e51b](https://github.com/Automattic/jetpack/pull/10333/commits/0f9e51b8f7ce9cfc1aecabcbdcefd88281d800fc)